admin/bench-measure.mk

```diff
@@ -35,10 +37,19 @@ measure: ./target/release/examples/bench
 	$^ handshake-resume TLS13_AES_256_GCM_SHA384
 	$^ handshake-ticket TLS13_AES_256_GCM_SHA384
 
-memory: ./target/release/examples/bench
+memory: $(BENCH)
 	$(MEMUSAGE) $^ memory TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 100
 	$(MEMUSAGE) $^ memory TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 1000
 	$(MEMUSAGE) $^ memory TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 5000
 	$(MEMUSAGE) $^ memory TLS13_AES_256_GCM_SHA384 100
 	$(MEMUSAGE) $^ memory TLS13_AES_256_GCM_SHA384 1000
 	$(MEMUSAGE) $^ memory TLS13_AES_256_GCM_SHA384 5000
+
+clean:
+	rm -f perf-*.svg
+	cargo clean
+
+$(BENCH):
+	cargo build --release -p rustls --example bench
```

Comment:
This should be `--profile=bench` rather than `--release` (see 134e43f9a38f875af3c3d3afb12bd29a3bb5bdbf for background).

Reply:
I've updated the makefile and the BENCHMARKING.md to reflect this comment